test/integration/adoption_application_reviews_test.rb

```diff
@@ -12,4 +12,12 @@ class AdoptionApplicationReviewsTest < ActionDispatch::IntegrationTest
       count: AdopterApplication.count, text: 'Edit Application'
     }
   end
+
+  test "unverified staff cannot access the page" do
+    sign_in users(:user_three)
+
+    get '/adopter_applications'
+
+    assert_response :redirect
```

Comment:
Could you please add an assertion to expect an alert flash after redirect? Check the application_controller.rb for the verified staff and assert that message in a flash. There are examples of this in the org_dogs_test.rb checking that unverified staff cannot access the org dogs page. I think it's handy to make sure the before action filter is working for this authorization.